### ISSUE TITLE
Notification Heads-up is not working, Followed all the solution in other threads, Nothing is working. 

### ISSUE THREAD

**Issue by jaysiddik** [OneSignal/react-native-onesignal#917]
Hi, 

Onesignal is in our react-native app (IOS and Android ) app. IOS is working fine with all the features that we expected. 

In android, Heads-up notification is not working, We have tried all the solution given in the GitHub community. 

Setting up the priority to 10 ! etc. Seems like none of the user has found a solution for this

Please help and head us in the right direction to fix this issue 

Thank you

**Comment by rgomezp** [comment]
Hello,
Please follow our issue template

**Comment by jkasten2** [comment]
@jaysiddik Heads-up notifications on Android 8.0+ require Android Categories / Channels to be setup and used on every notification. See OneSignal's documentation below and setup them up and using them.
https://documentation.onesignal.com/docs/android-notification-categories

**Comment by daheeahn** [comment]
I followed above link.
and I set importance to Urgent, then heads-up I really wanted is shown!

but, I have a question @rgomezp 
When I tried push alarm by New Message in OneSignal Dashboard at 12.20.2019, heads-up was shown without urgent of importance (I didn't set any android-notification-category). however, I tried same thing at today (12.28.2019) and heads-up is not shown without urgent of importance(android-notification-category).
What happened in a week?

@jkasten2 
Thank you very much!!!!
You saved my day :)
